src/action_creators/EnrollActionCreator.js

```diff
@@ -11,7 +12,50 @@ const EnrollActionCreator = {
     return {
       type: ActionTypes.TOGGLE_PREFERENCES
     }
+  },
+
+  requestStudentEnrollment(student) {
+    var that = this;
```

Comment:
There's no need to do that. `_receivedStudentEnrollmentSuccess` should be accessible.